src/test/javascript/spec/component/course/course.component.spec.ts

```diff
@@ -180,10 +180,7 @@ describe('CoursesComponent', () => {
     it('Should load next relevant exam', () => {
         const navigateSpy = sinon.spy(router, 'navigate');
         component.nextRelevantCourseForExam = course1;
-        component.nextRelevantExams = exams;
-        const findAllForDashboardStub = stub(courseService, 'findAllForDashboard');
-        findAllForDashboardStub.returns(of(new HttpResponse({ body: courses, headers: new HttpHeaders() })));
-
+        component.nextRelevantExams = [exam1];
```

Comment:
Why is this change necessary for this pull request?

Reply:
That was a flaky test case as discussed here: https://ls1tum.slack.com/archives/G6TCVL6HL/p1616406625032800?thread_ts=1616405571.030700&cid=G6TCVL6HL
So I fixed it together with Nicolas to get all tests passing again as demanded by Stephan https://github.com/ls1intum/Artemis/pull/3056#issuecomment-803630427